airflow/upgrade/rules/chain_between_dag_and_operator_not_allowed_rule.py

```diff
@@ -17,7 +17,7 @@
 from __future__ import absolute_import
 
 import re
-
+import os.path
```

Comment:
```suggestion
import os
```

We usually use `import os`